### ISSUE TITLE
Assert.xxxxx.Sequenceequals fails when comparing arrays contining null

### ISSUE THREAD

**Issue by FullValueRider** [twinbasic/twinbasic#1551]
An attempt at using assert to compare two arrays containing nulls fails.  The issue is demonstrated by the code below

```
Sub TestNullInArray()
       
        Dim myArray As Variant = Array(1, 2, 3, 4, 5)
        Dim myTest As Variant = Array(1, 2, 3, 4, 5)
        
        ' The assertion passes
        Assert.Strict.SequenceEquals myArray, myTest
        
        Dim myArray2 As Variant = Array(Null, Null, Null, 4, 5)
        Dim myTest2 As Variant = Array(Null, Null, Null, 4, 5)
        
        ' The next assertion fails
        Assert.Strict.SequenceEquals myArray2, myTest2
   End Sub
```

**Expected behavior**
The assertion should pass.

**Screenshots**
![ArrayWithNullsAssertionFail](https://user-images.githubusercontent.com/81257042/235445727-1179a477-bddb-47cf-bb9f-7cc8e53bc512.jpg)

**Desktop (please complete the following information):**
 - OS: [e.g. Windows 11]
 - twinBASIC compiler version Beta IDE 297

**Comment by fafalone** [comment]
I was curious about what Assert was so I started looking at it, so forgive me if this is a dumb comment, but the infotip for Assert.Strict says "Null is never considered equal to anything, not even itself"... so if Null <> Null, then isn't this the correct result?

**Comment by FullValueRider** [comment]
Well Yes and No. If Assert just did is null = null your observation would be correct.  However, VBA has the IsNull method for testing if a value is null so there is a mechanism which could be used in the assert code that tests for null in the same place in both sequences e.g. if IsNull(array1(index1)) then if isNull(array2(index1) then Is a match.  I appreciate that this is a bit of an edge case but there does appear to be way forward.  The test I present could have been written 
```
Assert.Strict.AreEqual IsNull(myTest(0), IsNull(myArray(0)
Assert.Strict.AreEqual IsNull(myTest(1), IsNull(myArray(1)

etc

```
So basically I'm being lazy and just trying to avoid excessive code in my test writing.
